src/crimson/osd/heartbeat.cc

```diff
@@ -309,6 +312,27 @@ seastar::future<> Heartbeat::handle_you_died()
   return seastar::now();
 }
 
+void Heartbeat::heartbeat_check()
+{
+  failure_queue_t failure_queue;
+  for (const auto& [osd, peer_info]: peers) {
+    if (clock::is_zero(peer_info.first_tx))
+      continue;
+
+    if (peer_info.is_unhealthy(clock::now())) {
+      logger().error(" heartbeat_check: no reply from osd.{} "
+		     "since back {} front {} (oldest deadline {})",
+		     osd, clock::to_time_t(peer_info.last_rx_back),
```

Comment:
no need to use `clock::to_time_t` for printing `ceph::coarse_real_clock::time_point`. please use `crimson/common/formatter.h` for access its formatter.